HW_1/task_4.py

```diff
@@ -0,0 +1,11 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, если разрешается сделать один разлом по
+# прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+n = int(input("Введите размер шоколадки (n):"))
+m = int(input("Введите размер шоколадки (m):"))
+k = int(input("Введите число долек (k):"))
+
+if k < (n * m) and ((k % n == 0) or (k % m == 0)):
```

Comment:
Решение верное, но некоторые скобки лишние.
```suggestion
if k < n * m and (k % n == 0 or k % m == 0):
```